<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetreDetailRecette</class>
 <widget class="QWidget" name="fenetreDetailRecette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#fenetreDetailRecette
{
	background-color: lightgreen;
	border:3px solid black;
	border-radius:15px;
}

QWidget#labelNom
{
	font-weight:bold;
	font-size: 20px;
	padding-left:0px;
	qproperty-alignment: AlignCenter;
}

QLabel
{
	font-size: 16px;
	padding-left:15 px;
}

strong
{

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelNom">
       <property name="text">
        <string>Nom Recette</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Durée  :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDuree">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Ingrédients nécessaires :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelIngredients">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Etapes de préparation :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelEtapes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
